index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>hw3-figma</title>
+</head>
+<body>
+    <header>
+        <div>
+            <img src="image/Group 31.png" alt="logo">
+            <img src="image/Productly.png" alt="text">
+        </div>
```

Comment:
зазвичай логотипи на сайтах є посиланнями на їх головну сторінку
```suggestion
        <a href="/">
            <img src="image/Group 31.png" alt="logo">
            <img src="image/Productly.png" alt="text">
        </a>
```